src/ORM/DB.php

```diff
@@ -133,7 +133,7 @@ public static function get_schema($name = 'default')
      * @param array $parameters Out parameter for the resulting query parameters
      * @param string $name An optional name given to a connection in the DB::setConn() call.  If omitted,
      * the default connection is returned.
-     * @return string The resulting SQL as a string
+     * @return string|null The resulting SQL as a string or null if there's no DB connector or the query was empty
```

Comment:
The method returns null under these conditions already - I was confused why I was getting null so had to go look. This will save other developers the same fate.